db/builder.cc

```diff
@@ -173,12 +173,28 @@ Status BuildTable(
             prev_key.assign(keys.front());
             ok = ParseInternalKey(Slice(prev_key), &prev_ikey);
             assert(ok);
+
+            // The front item has the smallest seqno in the merge queue
```

Comment:
Are you sure this is the case? Another comment in this same file says: "// seqno within the same key are in decreasing order"